packages/core/src/lib/light_push/index.ts

```diff
@@ -49,6 +50,11 @@ class LightPush extends BaseProtocol implements ILightPush {
     const recipients: PeerId[] = [];
 
     try {
+      if (!isSizeValid(message.payload)) {
+        log("Failed to send waku light push: message is bigger that 1MB");
+        return { recipients };
```

Comment:
Do we want to start return errors in this result?

Reply:
Yeah, good point. I'd still rather not throw, as hybrid let's put error codes there.